docs/Manual/03 Schedules.md

```diff
@@ -144,19 +146,19 @@ for i in range(16):
 ```
 
 #### Meaningless splits
-We describe Accera's behavior in a few degenerate cases. If the split size equals the dimension size, the transformation simply renames the split dimension. For example,
+We have described Accera’s behavior in a few degenerate cases. If the split size equals the dimension size, the transformation simply renames the split dimension. For example,
 ```python
 schedule = nest.create_schedule()
 kk = schedule.split(k, 11) # original size of dimension k was 11
 ```
 After the split, the size of `k` becomes 1 and the size of `kk` is `11`. The new shape of the iteration space is (16, 10, 1, 11). The dimension `k` becomes meaningless and therefore the schedule is basically unchanged.
 
-If the split size is greater than the dimension size, this is just a special case of the situation where the split size doesn't divide the dimension size. As mentioned above, Accera solves this by adding empty elements. For example,
+If* S* is greater than* D*, the split size doesn't divide the dimension size `D`. This special case is handled in Accera by simply adding empty elements. For example, 
 ```python
 schedule = nest.create_schedule()
 kk = schedule.split(k, 13)  # original size of dimension k was 11
 ```
-After the split, the size of `k` becomes 1 and the size of `kk` is `13`. The new shape of the iteration space is (16, 10, 1, 13), which means that 320 empty elements were added. These empty elements are removed during code generation, which means that the schedule is basically unchanged.
+After the split, the size of `k` becomes 1 and the size of `kk`, 13. The new shape of the iteration space is (16, 10, 1, 13), which means that 320 empty elements were added. These empty elements are removed during code generation, which shows that the schedule is basically unchanged.
```

Comment:
"which means that" is correct (we are not showing anything but just interpreting the schedule as unchanged under the covers)